src/main/resources/templates/editAthlete.html

```diff
@@ -6,17 +6,12 @@
 <title>View/Edit Athlete Schedule</title>
 </head>
 <body>
+
 	<h1>Edit Athlete Schedule Page</h1>
-	<a href="athleteList">
-		<button>Back</button>
-	</a>
-	</br>
-	</br>
-	<a href="index">
-		<button>Home</button>
-	</a>
-	</br>
-	</br>
+
+	<a class='link-button' href="athleteList">Back</a>
```

Comment:
Good suggestion... and it takes a keen eye to see what is going on here.

- Many developers use multiple br tags, where css margin or padding properties are better suited.

- So, good idea to remove br, and replace with css, which can vary the margin based on the user's viewing device.